dipy/tracking/local/tests/test_local_tracking.py

```diff
@@ -346,6 +350,68 @@ def allclose(x, y):
         npt.assert_(np.allclose(sl, expected[2]))
 
 
+def test_PeakDirectionTracker():
```

Comment:
PEP8: function name should be lowercase. I will be good if you can change all the test function name of this file :-)